qt/python/mantidqtinterfaces/mantidqtinterfaces/PyChop/PyChopGui.py

```diff
@@ -202,6 +198,18 @@ def setChopper(self, choppername):
         # Special case for MERLIN - only enable multirep for 'G' chopper
         self._merlin_chopper()
 
+    def _get_phase(self, key, widget):
+        idx = int(key[7])
```

Comment:
Since the only use of `key` in this function is `key[7]`, please can we just pass the int to the function, and give it a meaningful variable name? That would help a lot to clarify what it is.